data/campaigns/Winds_of_Fate/_main.cfg

```diff
@@ -0,0 +1,94 @@
+#textdomain wesnoth-wof
+
+[textdomain]
+    name="wesnoth-wof"
+[/textdomain]
+
+# wmlscope: set export=no
+[campaign]
+    id=Winds_of_Fate
+    name= _ "Winds of Fate"
+    abbrev= _ "WoF"
+    icon="units/drakes/inferno-fly-2-upstroke.png~CROP(0,0,72,72)~RC(magenta>red)"
+    background="story/landscape-mountains-05.jpg"
+    rank=152
+    start_year="3 YW"
+    end_year="4 YW"
+    define=CAMPAIGN_WINDS_OF_FATE
+    first_scenario="01_The_Hunt"
+    image="data/core/images/portraits/drakes/inferno.png~SCALE(350,350)"
+    description= _ "The story of how the drakes first arrived on the Great Continent.
+
+" + _ "(Hard level, 11 scenarios.)"
+
+    {CAMPAIGN_DIFFICULTY EASY      "units/drakes/burner-fly-1.png~CROP(0,0,72,72)~RC(magenta>red)"        ( _ "Aspirant")  ( _ "Normal")}
+    {CAMPAIGN_DIFFICULTY NORMAL    "units/drakes/fire-fire-se-1.png~RC(magenta>red)"                      ( _ "Intendant") ( _ "Challenging")} {DEFAULT_DIFFICULTY}
+    {CAMPAIGN_DIFFICULTY HARD      "units/drakes/inferno-fire-se-3.png~RC(magenta>red)"                   ( _ "Dominant")  ( _ "Difficult")}
+    {CAMPAIGN_DIFFICULTY NIGHTMARE "units/drakes/armageddon-melee-6.png~CROP(0,19,62,64)~RC(magenta>red)" ( _ "Ancestor")  ( _ "Nightmare")}
+
+    [about]
+        title= _ "Authors"
+        [entry]
+            name="Jonathan Kelly (name)"
+        [/entry]
+        [entry]
+            name = "Jeffrey 'Sigurd' Westcoat (SigurdFireDragon)"
+        [/entry]
+    [/about]
+    [about]
+        title = _ "Alpha Testing and Proofreading"
+        [entry]
+            name = "josteph"
+        [/entry]
+    [/about]
+    [about]
+        title= _ "Graphics"
+        [entry]
+            name="Lari Nieminen (zookeeper)"
+            comment="journey map"
+        [/entry]
+        [entry]
+            name="Emilien Rotival (LordBob)"
+            comment="story artwork"
+        [/entry]
+        [entry]
+            name="SkyOne"
+            comment="dolphin unit sprites and attack icon from Brave Wings campaign"
+        [/entry]
+        [entry]
+            name="L. Shelby"
+            comment="stymphalian & leopard unit sprites from Era of Four Moons & Panther Lord campaign"
+        [/entry]
+        [entry]
+            name="Vyncyn"
+            comment="caravel unit sprite from Rashy Era"
+        [/entry]
+    [/about]
+    [about]
+        title= _ "Based on Wings of Victory by:"
+        [entry]
+            name="Fabi/Fendrin"
+        [/entry]
+        [entry]
+            name="Eric S. Raymond (esr)"
+        [/entry]
+    [/about]
```

Comment:
Cuttle Fish only show up in the first scenario currently, but with both DW and WoF using Kraken, it will likely get moved to core, so this change would be good preparation for that. I will have to test out its effect on S1 and probably add it to the next version if no balance issues emerge.